spec.emu

```diff
@@ -281,13 +280,13 @@ contributors: Ujjwal Sharma, Younies Mahmoud
               1. Perform ! CreateDataPropertyOrThrow(_nfOpts_, `"minimumFractionDigits"`, _durationFormat_.[[FractionalDigits]]).
               1. Set _done_ to *true*.
           1. Else,
-            1. Set _value_ to the current value of the _valueSlot_ slot of _duration_.
+            1. Let _value_ be the current value of the _valueSlot_ slot of _duration_.
           1. If _style_ is `"2-digit"`, then
             1. Perform ! CreateDataPropertyOrThrow(_nfOpts_, `"minimumIntegerDigits"`, `2`).
-          1. If _value_ is *+0*<sub>𝔽</sub> or *-0*<sub>𝔽</sub> and _display_ is `"auto"`, then
+          1. If _value_ is 0 and _display_ is `"auto"`, then
             1. Skip to the next iteration.
           1. Let _nf_ be ? Construct(%NumberFormat%, &laquo; _durationFormat_.[[Locale]], _nfOpts_ &raquo;).
-          1. Let _num_ be ! FormatNumeric(_durationFormat_.[[NumberFormat]], _value_).
+          1. Let _num_ be ! FormatNumeric(_durationFormat_.[[NumberFormat]], 𝔽(_value_)).
```

Comment:
`𝔽` will need to be removed after `Intl.NumberFormat` V3 lands. But this will lead to a different output for some inputs, so if we care about the exact same output, this line will need to be changed to `𝔽(ℝ(value))`. 

Or alternatively we could make `Intl.DurationFormat` dependent on `Intl.NumberFormat` V3.

I guess it's better to open a separate issue about the precision, because `Temporal` typically generates the precise result.

Reply:
Ping @sffc. How do you think we should proceed?

Reply:
#110 has a concrete example to show the possible precision loss.